week-1/K-functions-parameters/exercise5.js

```diff
@@ -3,6 +3,6 @@ function createLongGreeting(name, age) {
   return "Hello, my name is " + name + " and I'm " + age + " years old.";
 }
 
-const greeting = createLongGreeting("Daniel", 30);
+var greeting = createLongGreeting("Daniel", 30);
```

Comment:
This isn't wrong, and you are correct that we are just looking at the `var` keyword in class at the moment. In the future classes we will cover the `const` and `let` keywords. So you can leave it as it is if you want.